terra_notebook_utils/drs.py

```diff
@@ -236,12 +244,16 @@ def _do_copy_drs(drs_uri: str,
                  indicator_type: Indicator,
                  workspace_name: Optional[str]=WORKSPACE_NAME,
                  workspace_namespace: Optional[str]=WORKSPACE_GOOGLE_PROJECT):
-    dst_blob: Union[GSBlob, URLBlob, LocalBlob]
+    dst_blob: Union[GSBlob, URLBlob, LocalBlob, AzureBlob]
     src_info = get_drs_info(drs_uri)
     src_blob = get_drs_blob(src_info, workspace_namespace)
     if dst.startswith("gs://"):
         bucket_name, key = _resolve_bucket_target(dst, src_info)
         dst_blob = GSBlob(bucket_name, key)
+    # Azure url looks like https://qijlbdgpc4zqdee.blob.core.windows.net/qi-test-container/subdir/another/qi-blob3
+    if "windows.net" in dst:
```

Comment:
I feel like, ideally, we'd have a wasb:// scheme to distiguish these, especially if we want plain https:// support in the future.

Would you consider making this stricter?  If so, would something like:

```
if url.startswth('https://') and url[len('https://'):].split('/')[0].endswith('.blob.core.windows.net'):
    ...
```

or even a function like:


```
def is_azure_url(self, azure_url: str) -> bool:
    try:
        self._resolve_azure_blob_path(azure_url)
        return True
    except:
        return False

if is_azure_url(uri):
    ...
```

Work?